js/components/developer/networking-screen/webAppCompany.js

```diff
@@ -34,26 +35,20 @@ export default class WebAppCompany extends Component {
 
   // Fetch data from this.props.url and update state
   fetchData() {
-    fetch(this.props.url)
-      .then((response) => {
-        if (response.status === 200) {
-          return response.json();
-        }
-        throw new Error('Response was not 200 OK!');
-      })
-      .then((responseJson) => {
+    getJson(this.props.url,
+      (responseJson) => {
         this.setState({
           data: responseJson.data,
           loading: false,
         });
-      }).catch((error) => {
+      },
+      (error) => {
```

Comment:
There are some more information (just deployed it) about error codes returned from the API that you can review here: https://sandbox-api.justarrived.xyz/api_docs#http_codes.